internal/web/server.go

```diff
@@ -0,0 +1,179 @@
+package web
+
+import (
+	"context"
+	"embed"
+	"fmt"
+	"html/template"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/styrainc/regal/internal/explorer"
+	"github.com/styrainc/regal/internal/lsp/cache"
+	"github.com/styrainc/regal/internal/lsp/clients"
+)
+
+//go:embed assets/*
+var assets embed.FS
+
+//nolint:gochecknoglobals,lll
+var tpl = template.Must(template.New("main").Parse(`
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8"/>
+		<link rel="stylesheet" href="/assets/missing.min.css">
```

Comment:
It looks like the missing.css file in the PR is unused? only the min.css is referenced here.

Reply:
Ah yes, I guess we need the other one if we ever want to change anything? But we can always get it from @srenatus 😄

Reply:
It's not mine, it's https://missing.style. -- @anderseknert had copied it into the opa-explorer repo at some point when they had changed their URLs or had an outage or something. We could go back to using their import instead and not copy any css here:

```html
<link rel="stylesheet" href="https://unpkg.com/missing.css@1.1.3">
```